static/templates/left_sidebar.hbs

```diff
@@ -55,26 +55,23 @@
                 <span class="arrow drafts-sidebar-menu-icon"><i class="zulip-icon zulip-icon-ellipsis-v-solid" aria-hidden="true"></i></span>
             </li>
         </ul>
+    </div>
 
     <div id="pm_stream_scroll_container" class="scrolling_list" data-simplebar>
         <div id="private_messages_section" class="zoom-out hidden-for-spectators">
             <div id="private_messages_list">
                 <div id="private_messages_section_header" class="zoom-out zoom-in-sticky">
-                    <div class="left_part">
-                        <a class="hide_more_private_messages zoom-out-hide">
-                            <i class="fa fa-chevron-left" aria-hidden="true"></i>
-                            {{t 'BACK' }}
-                        </a>
-                        <i id="toggle_private_messages_section_icon" class="fa fa-lg fa-caret-down toggle_private_messages_section zoom-in-hide" aria-hidden="true"></i>
-                        <span id="more_private_messages_sidebar_title" class="zoom-out-hide">{{t 'Private messages' }}</span>
-                        <h4 class="sidebar-title toggle_private_messages_section zoom-in-hide" data-tippy-content="{{t 'Private messages' }}">{{t 'PRIVATE MESSAGES' }}</h4>
-                    </div>
-                    <div class="right_part">
-                        <span class="unread_count"></span>
-                        <a href="#narrow/is/private" id="show_all_private_messages">
-                            <i class="fa fa-align-right" aria-hidden="true"></i>
-                        </a>
-                    </div>
+                    <a class="hide_more_private_messages zoom-out-hide">
+                        <i class="fa fa-chevron-left" aria-hidden="true"></i>
+                        {{t 'BACK' }}
+                    </a>
+                    <span id="more_private_messages_sidebar_title" class="zoom-out-hide">{{t 'Private messages' }}</span>
+                    <i id="toggle_private_messages_section_icon" class="fa fa-lg fa-caret-down toggle_private_messages_section zoom-in-hide" aria-hidden="true"></i>
+                    <h4 class="sidebar-title toggle_private_messages_section zoom-in-hide" data-tippy-content="{{t 'Private messages' }}">{{t 'PRIVATE MESSAGES' }}</h4>
+                    <span class="unread_count"></span>
+                    <a href="#narrow/is/private" id="show_all_private_messages">
+                        <i class="fa fa-align-right" aria-hidden="true"></i>
+                    </a>
```

Comment:
I had to re-arrange the left-sidebar structure back to how I committed it in my previous commit as the new structure proposed by you @amanagr  would break the ZoomedIn view -- since it has `< BACK` in a separate line and `Private messages` with `All private messages` view icon in a separate one whereas the only visible benefit of this structure was to put them in a single line which is not what we want.

the previous view -> 
![image](https://user-images.githubusercontent.com/76561593/187785720-8c2ecabf-54e0-4796-ba78-87f8d82e1dbf.png)

and what we actually want -> 
![Uploading image.png…]()